contrib/runners/mistral/runner.yaml

```diff
@@ -2,8 +2,8 @@
   description: A runner for executing mistral v2 workflow.
   enabled: true
   name: mistral-v2
-  query_module: st2actions.query.mistral.v2
-  runner_module: st2actions.runners.mistral.v2
+  query_module: mistralv2
+  runner_module: mistralv2
```

Comment:
It doesn't currently support sub-directories because the modules are loaded by `<base_path>/runners/<runner_module>/<runner_module>.py`. We could change it to use `<base_path>/runners/<runner_name>/<runner_module>.py`

What is the goal? Is the goal to load various modules from the same folder or to be able to have sub-directories inside of runners? For instance `<base_path>/runners/<runner_name>/<some>/<new>/<path>/<runner_module>.py`?